client/src/App.jsx

```diff
@@ -1,13 +1,132 @@
 import './App.css'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { useEffect, useState } from 'react';
+import WildlifeObservations from './components/WildlifeObservations';
+import Species from './components/Species';
+import IndividualsPopup from './components/IndividualsPopup';
+import SightingsPopup from './components/SightingsPopup';
+import { Link } from 'react-router-dom';
 
 
 function App() {
 
+  const [observation, setObservation] = useState([]);
+
+  const [show, setShow] = useState({
+    "species" : false,
+    "individuals" : false,
+    "sightings" : false
+  });
+
+  const handleClose = (type) => {
```

Comment:
@beyzanurck Not anything big... But you create one function for handlleClose and handleShow and just pass true or false as parameter.

They use the same logic except for bool in it. Try to always think DRY.

Reply:
@esolar07 I combined them into the handleVisibility function. Thank you!